src/SoftApService.js

```diff
@@ -94,22 +107,34 @@ class SoftAPService {
     return scans.map(translateWifiFromApi);
   };
 
-  static _getPublicKey = async (): Promise<Object> => {
-    const { b: rawDerPublicKey, r: responseCode } = await fetchJSON(
-      `${BASE_URL}/public-key`,
-    );
-
-    if (responseCode !== SUCCESS_RESPONSE_CODE) {
-      throw new Error('Error on getting public Brewskey box key!');
-    }
-
-    const derBuffer = Buffer.from(rawDerPublicKey, 'hex');
-    const publicKey = new NodeRSA(derBuffer.slice(22), 'pkcs1-public-der', {
-      encryptionScheme: 'pkcs1',
+  static _getPublicKey = async (): Promise<Object> =>
+    new Promise(async (resolve, reject) => {
+      setTimeout(
+        () =>
+          reject(
+            new Error(
```

Comment:
this more shitty case, because the only way to make network responsible again is poweroff/poweron device and repeat the whole setup process again. fortunately, this happens pretty rarely.